cmd/outline-ss-server/main.go

```diff
@@ -92,11 +92,7 @@ func (s *SSServer) startPort(portNum int) error {
 	packetHandler := service.NewPacketHandler(s.natTimeout, port.cipherList, s.m)
 	s.ports[portNum] = port
 	accept := func() (transport.StreamConn, error) {
-		conn, err := listener.AcceptTCP()
-		if err == nil {
-			conn.SetKeepAlive(true)
-		}
-		return conn, err
+		return listener.AcceptTCP()
 	}
 	go service.StreamServe(accept, tcpHandler.Handle)
```

Comment:
Does this not work?

```go
go service.StreamServe(listener.AcceptTCP, tcpHandler.Handle)
```

Reply:
It needs to be wrapped because of `StreamServe`'s signature. I just retried using `WrapStreamListener` though and that seems to work now. Not sure why that didn't work earlier.